RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py

```diff
@@ -67,449 +67,50 @@
-
-    #Loose Id
-    Pt010_BetaStarLoose    = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt1020_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt2030_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt3040_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt4050_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-
-    #RMS variable
-    #Tight Id
-    Pt010_RMSTight         = cms.vdouble( 0.06, 0.07, 0.04, 0.05),
-    Pt1020_RMSTight        = cms.vdouble( 0.06, 0.07, 0.04, 0.05),
-    Pt2030_RMSTight        = cms.vdouble( 0.05, 0.07, 0.03, 0.045),
-    Pt3040_RMSTight        = cms.vdouble( 0.05, 0.06, 0.03, 0.04),
-    Pt4050_RMSTight        = cms.vdouble( 0.05, 0.06, 0.03, 0.04),
-    
-    #Medium Id => Daniele
-    Pt010_RMSMedium        = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt1020_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt2030_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt3040_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt4050_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-
-    #Loose Id
-    Pt010_RMSLoose         = cms.vdouble( 0.06, 0.05, 0.05, 0.07),
-    Pt1020_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.07),
-    Pt2030_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055),
-    Pt3040_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055),
-    Pt4050_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055)
-    )
-
-
-JetIdParams = cms.PSet(
-    #4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
-
-    #Tight Id
-    Pt010_Tight    = cms.vdouble( 0.5,0.6,0.6,0.9),
-    Pt1020_Tight   = cms.vdouble(-0.2,0.2,0.2,0.6),
-    Pt2030_Tight   = cms.vdouble( 0.3,0.4,0.7,0.8),
-    Pt3040_Tight   = cms.vdouble( 0.5,0.4,0.8,0.9),
-    Pt4050_Tight   = cms.vdouble( 0.5,0.4,0.8,0.9),
-
-    #Medium Id
-    Pt010_Medium   = cms.vdouble( 0.2,0.4,0.2,0.6),
-    Pt1020_Medium  = cms.vdouble(-0.3,0. ,0. ,0.5),
-    Pt2030_Medium  = cms.vdouble( 0.2,0.2,0.5,0.7),
-    Pt3040_Medium  = cms.vdouble( 0.3,0.2,0.7,0.8),
-    Pt4050_Medium  = cms.vdouble( 0.3,0.2,0.7,0.8),
-
-    #Loose Id
-    Pt010_Loose    = cms.vdouble( 0. , 0. , 0. ,0.2),
-    Pt1020_Loose   = cms.vdouble(-0.4,-0.4,-0.4,0.4),
-    Pt2030_Loose   = cms.vdouble( 0. , 0. , 0.2,0.6),
-    Pt3040_Loose   = cms.vdouble( 0. , 0. , 0.6,0.2),
-    Pt4050_Loose   = cms.vdouble( 0. , 0. , 0.6,0.2)
+##########################################################
+## Empty cutbased WP for compatibility
+###########################################################
+EmptyCutBased_wp = cms.PSet(
+    #4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
+
+    Pt010_BetaStarTight    = cms.vdouble(-999.,-999.,-999.,-999.),
+    Pt1020_BetaStarTight   = cms.vdouble(-999.,-999.,-999.,-999.),
+    Pt2030_BetaStarTight   = cms.vdouble(-999.,-999.,-999.,-999.),
+    Pt3040_BetaStarTight   = cms.vdouble(-999.,-999.,-999.,-999.),
+    Pt4050_BetaStarTight   = cms.vdouble(-999.,-999.,-999.,-999.),
```

Comment:
```suggestion
EmptyCutBased_wp = cms.PSet()
#4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
for pt in ['010', '1020', '2030', '3040', '4050']:
  for tp in ['BetaStar', 'RMS']:
    for wp in ['Loose', 'Medium', 'Tight']:
      setattr(EmptyCutBased_wp, "Pt"+pt+"_"+tp+wp, cms.vdouble(-999.,-999.,-999.,-999.))
```
seems like a more compact way to write down the 30-some lines of the definition of this "dummy" PSet

Reply:
thanks for the suggestion.